<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
  <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
<style>
.button {
    background-color: #4CAF50;
    border: none;
    color: white;
    padding: 15px 15px;
    border-radius: 16px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 18px;
    margin: 4px 2px;
}
</style>
</head>
<body>
    <div class="row" style="margin-right: 50px; margin-left: 50px; padding-top: 50px;">
        <div class="col-md-4">
            <div class="w3-container w3-blue w3-center">
                <h3>About ADACT</h3>
            </div>
            <div class="w3-container">
                ADACT is a completely free, open source sequence comparison tool which 
                measures disimilarities among several species based on alignment-free mannar. ADACT takes several genome 
                sequences and some configuration parameters (e.g. sequence input type, K-mer size, absent word type, disimilarity index, RC-setting) 
                as input. On the other hand, it outputs distance matrix, sorted species relationship and phylogenetic tree among species.
            </div>
        </div>
        <div class="col-md-4">
            <div class="w3-container w3-blue w3-center">
                <h3>How to start</h3>
            </div>
            <div class="w3-container">
                If a bioinformatician wants to run a test in ADACT he must create an account on ADACT otherwise 
                he must use his existing account. After starting a run on ADACT, he can close the tab even shut down his pc.
                His project will automatically run on our server without further assistance from user. After successful
                completion of project, notification will be sent via email to the user. The project's data will be temporarily stored 
                on our server. Besides, project's metadata will be permanently saved.
            </div>
        </div>
        <div class="col-md-4">
            <div class="w3-container w3-blue w3-center">
                <h3>Accepted Input type</h3>
            </div>
            <div class="w3-container">
                One bioinformatician can input genome sequences in fasta format. In fasta format, ADACT supports 
                <b>single sequence single fasta</b> (one genome sequence in a fasta file) and <b>multiple 
                sequence single fasta</b> (several genome sequences in a fasta file) file. Many bioinformaticians
                are ease with accession number and GI number. For their convenience, ADACT provides an option to input 
                sequences as accession number and GI number.
            </div>
        </div>
    </div>
    <div class="row" style="margin-right: 50px; margin-left: 50px; padding-top: 50px;">
        <div class="col-md-4">
            <div class="w3-container w3-blue w3-center">
                <h3>Configuration Parameters</h3>
            </div>
            <div class="w3-container">
                The following parameters should be provided when a project is started. 
                <ul>
                    <li>Input sequence type</li>
                    <li>Short name</li>
                    <li>Absent word type</li>
                    <li>K-mer size</li>
                    <li>RC-type</li>
                    <li>Genome sequence type</li>
                    <li>Disimilarity Index</li>
                </ul>  
            </div>
        </div>
    
        <div class="col-md-4">
            <div class="w3-container w3-blue w3-center">
                <h3>Disimilarity Index</h3>
            </div>
            <div class="w3-container">
                Disimilarities among species are measured according to following metrics 
                <ul>
                    <li>Length weighted index (both on MAW and RAW)</li>
                    <li>Jaccard Distance (only on MAW)</li>
                    <li>Total Variation Distance (only on MAW)</li>
                    <li>GC content (both on MAW and RAW)</li>
                </ul>
                To read brief description <a href="https://bmcresnotes.biomedcentral.com/articles/10.1186/s13104-016-1972-z">click here</a>.
  
            </div>
        </div>
        <div class="col-md-4">
            <div class="w3-container w3-blue w3-center">
                <h3>What ADACT generates</h3>
            </div>
            <div class="w3-container">
                The following outputs are generated by ADACT after a successful run 
                <ul>
                    <li>Distance matrix of species</li>
                    <li>Sorted species relation</li>
                    <li>UPGMA (Unweighted Pair Group Method with Arithmetic Mean) tree</li>
                    <li>NJ (Neighbour Joining) tree</li>
                </ul>
            </div>
        </div>
    </div>
    <center><h1>Terminologies in ADACT</h1></center>

    <ul class="nav nav-tabs" style="margin: 50px;">
        <li class="active"><a data-toggle="tab" href="#maw">MAW</a></li>
        <li><a data-toggle="tab" href="#raw">RAW</a></li>
        <li><a data-toggle="tab" href="#lwi">LWI</a></li>
        <li><a data-toggle="tab" href="#jaccard">Jaccard</a></li>
        <li><a data-toggle="tab" href="#tvd">TVD</a></li>
        <li><a data-toggle="tab" href="#gcc">GCC</a></li>
        <li><a data-toggle="tab" href="#acc">Accession number</a></li>
        <li><a data-toggle="tab" href="#gi">GI number</a></li>
        <li><a data-toggle="tab" href="#rc">RC</a></li>
        <li><a data-toggle="tab" href="#short">Short name</a></li>
    </ul>

    <div class="tab-content" style="margin: 50px;">
        <div id="maw" class="tab-pane fade in active">
            <h3>MAW</h3>
            <p>A Minimal Absent Word of a sequence is an absent word whose proper factors (longest prefix and longest suffix) 
            all occur in the sequence. More formally, string <i>y</i> will be an absent word of string <i>x</i> if <i>y</i> is not a factor 
            of string <i>x</i> but all proper factors of <i>y</i> is present in string <i>x</i>.</p>
        </div>
        <div id="raw" class="tab-pane fade">
            <h3>RAW</h3>
            <p>Relative absent of string <i>x</i> with respect to string <i>y</i> are the minimal strings that can't be found in <i>x</i> 
            but are present in <i>y</i>.</p>
        </div>
        <div id="lwi" class="tab-pane fade">
            <h3>Length Weighted Index</h3>
            <p>The length Weighted Index (LWI) provides a measure of the similarity/dissimilarity of two sets by considering the length of 
                each member in the symmetric difference or intersection of these sets.</p>
        </div>
        <div id="jaccard" class="tab-pane fade">
            <h3>Jaccard Distance</h3>
            <p>Jaccard Distance index is a statistical measure to use as a similarity coefficient between sample sets. Jaccard Distance 
            is evaluated as the ratio of MAWs uncommon between two strings.</p>
        </div>
        <div id="tvd" class="tab-pane fade">
            <h3>Total Variation Distance</h3>
            <p>Total Variation Distance (TVD) is used to assess pairwise variance. To calculate TVD between two sequences <i>x</i> and <i>y</i>, we 
            first calculate the number of MAWs in <i>MAW<sub>x</sub></i> and <i>MAW<sub>y</sub></i>  for each word length and then convert this histogram to a normalized 
            version that can be explained as a probability distribution.
            </p>
        </div>
        <div id="gcc" class="tab-pane fade">
            <h3>GC Content</h3>
            <p>This index is based on the content of MAW sets not on number statistics of the MAW sets.  
            In particular, it focuses on the compositional bias or GC Content  (overall fraction of G 
            plus C nucleotides) of the MAW sets. GC Content is calculated considering both symmetric difference and intersection on MAW sets. 
            </p>
        </div>
        <div id="acc" class="tab-pane fade">
            <h3>Accession number</h3>
            <p>In bioinformatics, an accession number is an unique identifier assigned to each DNA or protein sequence record to ease for following of different versions of
             that sequence record and the associated sequence over time in a single data repository such as NCBI.
            </p>
        </div>
        <div id="gi" class="tab-pane fade">
            <h3>GI number</h3>
            <p>A GI number (for GenInfo Identifier, normally written in lower case, "gi") is an arbitrary sequence of numbers which are allowed consecutively to all sequence 
            record gathered by National Center for Biotechnology Information (NCBI). The GI number carries no information about the version number of the sequence record.
            </p>
        </div>
        <div id="rc" class="tab-pane fade">
            <h3>Reverse Complement</h3>
            <p>The reverse complement of a DNA sequence is formed by reversing the letters and then swapping A and T and swapping C and G. Thus the reverse complement 
            of string "ACCTGAG" is "CTCAGGT".
            </p>
        </div>
        <div id="short" class="tab-pane fade">
            <h3>Short name</h3>
            <p>We provide the user the option to choose short names according to their will. It must be short enough to ensure better-looking short phylogenetic trees.
            </p>
        </div>
        
    </div>

    <center><h1> ADACT Development Team</h1></center>
    <div class="row">
        <div class="col-md-4 w3-center">
            <img src="../../../public/profile/mujtahid.jpg" 
                    style='height: 40%; width: 40%; object-fit: contain;'>
            <h5> Mujtahid Akon </h5>
        </div>
        <div class="col-md-4 w3-center">
            <img src="../../../public/profile/mahi.jpg" 
                    style='height: 40%; width: 40%; object-fit: contain; border-radius: 20%;' >
            <h5> Mohimenul Kabir </h5>
        </div>
        <div class="col-md-4 w3-center">
            <img src="../../../public/profile/muntasir.jpg" 
                    style='height: 40%; width: 40%; object-fit: contain;' >
                  <h5> Muntashir Akon </h5>
        </div>
    </div>
    <div class="row">
        <div class="col-md-2 w3-center">
        </div>
        <div class="col-md-4 w3-center">
            <img src="../../../public/profile/avatar.jpg" 
                    style='height: 40%; width: 40%; object-fit: contain;' >
        </div>
        <div class="col-md-4 w3-center">
            <img src="../../../public/profile/avatar.jpg" 
                    style='height: 40%; width: 40%; object-fit: contain; border-radius: 20%;' >
        </div>
        <div class="col-md-2 w3-center">
        </div>
    </div>
</body>
